test/functional/tests/security/fuzzy/config/device.yml

```diff
@@ -0,0 +1,14 @@
+- name: String
+  attributes:
+    name: Device
+    value: '<DEV>'
+    mutable: 'true'
+  children:
+   - name: Hint
+     attributes:
+       name: type
+       value: 'path'
+   - name: Hint
+     attributes:
+       name: ValidValues
+       value: ''
```

Comment:
Some of these configs need parametrization, which generates some amount of ugly looking code in the test. How about encapsulating the schema + parametrization within a helper function?